### ISSUE TITLE
Unable to open Touch Keyboard after update

### ISSUE THREAD

**Issue by clover-yan** [ramensoftware/windhawk#126]
Unable to open Touch Keyboard after update. I just updated Windhawk and I cannot turn Touch Keyboard on. By using Safe Mode I can use Touch Keyboard.

No help to disable all mods. Seems like issue of Windhawk.

**Comment by clover-yan** [comment]
Some Experience Pack features also became unusable, e.g. Snipping Tool.

**Comment by Eussora** [comment]
Neither the Search shortcut (Win+S) works. Already tested with both 1.3.1 and 1.4. Only the latest has this problem.

**Comment by Aleksanderis** [comment]
> I wasn't able to reproduce the issue. Perhaps those tools either crash or are being killed, for example by an antivirus. Are you using an antivirus? Can you check Event Viewer for crashes? You can use this guide: https://www.cyberlink.com/support/faq-content.do?id=10449

Not sure about which tools you are speaking and how it's related to CyberLink, your link also redirects me to CyberLink support center (https://www.cyberlink.com/support-center/home). However I checked Event Viewer and see following errors there (which are repeated several times):

> Faulting application name: SearchHost.exe, version: 623.22800.10.0, time stamp: 0x6514fd71
> Faulting module name: windhawk.dll, version: 1.4.0.0, time stamp: 0x6546cb8d
> Exception code: 0xc0000409
> Fault offset: 0x000000000003c689
> Faulting process id: 0x0x40DC
> Faulting application start time: 0x0x1DA118C05742EE5
> Faulting application path: C:\Windows\SystemApps\MicrosoftWindows.Client.CBS_cw5n1h2txyewy\SearchHost.exe
> Faulting module path: C:\Progs\Windhawk\Engine\1.4\64\windhawk.dll
> Report Id: b69a6946-a938-432b-977b-10705510ba3e
> Faulting package full name: MicrosoftWindows.Client.CBS_1000.22674.1000.0_x64__cw5n1h2txyewy
> Faulting package-relative application ID: CortanaUI

**Comment by m417z** [comment]
> Your link also redirects me to CyberLink support center

Yeah, looks like they removed that page, here's an archived copy:
https://web.archive.org/web/20230525052835/https://www.cyberlink.com/support/faq-content.do?id=10449

> However I checked Event Viewer and see following errors there, which are repeated several times and which seems relevant.

Indeed likely to be relevant. Can you capture a crash dump? You should be able to get it with the following steps:
* Open regedit
* Go to: `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting`
* Create a key named `LocalDumps`
* Create a DWORD value named `DumpType` with value `2`
* Trigger the crash
* Go to the `%LocalAppData%\CrashDumps` folder, you should see a dump file in there

**Comment by Aleksanderis** [comment]
I've got a crash dump, but they are 600+ Mb each..
Uploaded them to Google Drive, but I don't want to share them with everyone. How could I share it with you privately?

**Comment by clover-yan** [comment]
Before we can get it fixed, can we convert the portable version to installed version? If so, how to?

**Comment by m417z** [comment]
Fixed in Windhawk v1.4.1.

**Comment by Aleksanderis** [comment]
Tested it and eventually it worked. But just a note for others - don't be scared if it doesn't work at first, like it was for me. :)

I installed an updated portable version, started it - "start menu" was working, but things like "win+." (emoji dialog, TextInputHost.exe) - wasn't working. Then I looked for logs in Event Viewer and noticed there were crashes logged, which were saying Windhawk 1.4.0 (not 1.4.1). I'm not sure if 1.4.1 reveals itself as 1.4.1 or 1.4.0 (since it's a minor version change), but it gave a hint anyway, that old module could be still injected somewhere. So I Restarted explorer.exe, etc - and it started to work.

Great work, @m417z! 🚀
